sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/PTransform.java

```diff
@@ -176,8 +176,11 @@
 public abstract class PTransform<InputT extends PInput, OutputT extends POutput>
     implements Serializable /* See the note above */, HasDisplayData {
   /**
-   * Applies this {@code PTransform} on the given {@code InputT}, and returns its
-   * {@code Output}.
+   * Override this method to specify how this {@code PTransform} should be expanded
+   * on the given {@code InputT}.
+   *
+   * <p>NOTE: This method should never be called. Instead, the {@code PTransform}
```

Comment:
"should not be called directly. Instead, apply the transform to the {@link PInput input} using..."